lib/main.dart

```diff
@@ -166,46 +150,49 @@ class MainPage extends StatelessWidget {
         builder: (context) {
           return AlertDialog(
             title: Text('Select album'),
-            content: ListView.builder(
-                itemCount: Albums.getAlbumList(context).length,
-                itemBuilder: (BuildContext ctxt, int index) {
-                  return GestureDetector(
-                      onTap: () {
-                        addPhotosToAlbum(
-                            Albums.getAlbumList(context)[index].id, context);
-                      },
-                      child: Card(
-                          child: ListTile(
-                              title: Text(
-                                  Albums.getAlbumList(context)[index].name))));
-                }),
+            content: Container(
+              width: double.maxFinite,
```

Comment:
@n1try as I don`t have much experience in Flutter, I have one small question regarding the change: what is the effect of wrapping the ListView with a Container and setting the width to `double.maxFinite`?

Reply:
To be honest I essentially searched for the error message and found [this fix](https://stackoverflow.com/a/56355962/3112139). Without being too sufficient in Flutter, my bare understanding is that when not wrapping the list inside a container, the renderer can't know the lists horizontal and vertical dimensions before rendering each and every item, which is costly. The container provides kind of a constrained frame around the potentially endlessly large list. I can't particularly explain why setting the width to effectively infinite is helpful. I would assume that setting it to the actual screen width would have the same effect.

Reply:
ok, this seems to make sense, it's interesting that the problem only occurs on some devices.
Thanks, for fixing it :+1:

Reply:
It's only my understanding, no guarantee for correctness :wink: 
According to the post I shared above, it's exactly all phones with Android < 9.0 that cause the problem.